### DISCUSSION TITLE
[疑似BUG]在自定义主页教学文件中如何使用Foreground属性

### DISCUSSION THREAD

**Discussion by KOFTWAR** [Meloong-Git/PCL#357] [提问] [answered]
自定义主页教学文件中显示Foreground属性是通用的，但是在local:MyButton项中却无法使用这一属性，我在TextBlock项下尝试使用了同样的属性参数，这是可用的
PCL2提示无法设置位置成员local:MyButton.Foreground

**Comment by LTCatt** [comment, marked as answer]
MyButton 只允许调整主题。
描述上确实有误差，可以去帮助库交个 issue。
